### PR TITLE
feat(gallery): use current page index to query assets from imgix

### PR DESCRIPTION
**Pull request by sherwinski**
This PR allows users to select a page of assets based on the list items available in the pagination dropdown menu.

https://user-images.githubusercontent.com/15919091/125862924-05aa67e5-cf9c-409f-8dac-d986ee6832ab.mov

### CODE REVIEW

File: src/components/Gallery/ImagePagination.tsx [hunk 1]
```diff
@@ -47,7 +47,10 @@ export function ImagePagination({
             indicateDropdown
             onClick={() => setOpen(!isOpen)}
           >
-            {'Page ' + pageInfo.current + ' of ' + pageInfo.totalPageCount}
+            {'Page ' +
+              `${pageInfo.currentIndex + 1}` +
+              ' of ' +
+              pageInfo.totalPageCount}
```
Comment (luqven) [inline src/components/Gallery/ImagePagination.tsx:53]:
```suggestion
            {`Page ${pageInfo.currentIndex + 1} of ${pageInfo.totalPageCount}`}
```

**Review by luqven — APPROVED** [review]
LGTM 👍🏼

File: src/components/Gallery/ImagePagination.tsx (hunk 1 above)
Comment (sherwinski) [reply src/components/Gallery/ImagePagination.tsx:53]: Unfortunately, this is enforced by the linter 🙁

Comment (sherwinski) [reply src/components/Gallery/ImagePagination.tsx]: good catch 😅

**Pull request merged**